packages/compass-e2e-tests/tests/intercom.test.ts

```diff
@@ -0,0 +1,44 @@
+import { init, cleanup, screenshotIfFailed } from '../helpers/compass';
+import type { Compass } from '../helpers/compass';
+
+describe('Intercom integration', function () {
+  let compass: Compass;
+
+  before(async function () {
+    compass = await init(this.test?.fullTitle(), { firstRun: true });
+  });
+
+  afterEach(async function () {
+    await screenshotIfFailed(compass, this.currentTest);
+  });
+
+  after(async function () {
+    // clean up if it failed during the before hook
+    await cleanup(compass);
+  });
+
+  it('should load Intercom script', async function () {
```

Comment:
@addaleax does this test makes sense to you? The only concern would be this flaking because loading the script failed, but I'm pretty certain it should be relatively stable. If it becomes an issue, we can always remove it and think of a different way to test this.

Reply:
Yes, it does :)